src/dropdowns/page-click.service.ts

```diff
@@ -0,0 +1,19 @@
+import {Injectable, EventEmitter} from 'angular2/core';
+
+@Injectable()
+export class NglPageClickService {
```

Comment:
Can we avoid this by subscribing `Renderer.listenGlobal` on each dropdown when it's opening, and unsubscribe when it's closing?